static/app/views/performance/transactionSummary/transactionOverview/eapSidebarCharts.tsx

```diff
@@ -54,20 +59,50 @@ function FailureRateWidget({transactionName}: FailureRateWidgetProps) {
     REFERRER
   );
 
-  if (isFailureRatePending || isFailureRateError) {
+  const {
+    data: failureRateValue,
+    isPending: isFailureRateValuePending,
+    isError: isFailureRateValueError,
+  } = useEAPSpans(
+    {
+      search: new MutableSearch(`transaction:${transactionName}`),
+      fields: ['failure_rate()'],
+      pageFilters: selection,
+    },
+    REFERRER,
+    true
+  );
+
+  const getFailureRateBadge = () => {
+    if (isFailureRateValuePending || isFailureRateValueError) {
+      return null;
+    }
+
+    const failureRateText = toPercent(failureRateValue[0]?.['failure_rate()'] ?? 0);
+    return (
+      <Tag key="failure-rate-value" type="error">
+        {failureRateText}
+      </Tag>
+    );
+  };
```

Comment:
I think you should skip doing `toPercent` on your own, and use `getFieldRenderer` instead. That way it'll be formatted the same way as everywhere. That code is a little wacky though, so I think you'll end up wanting to put this into a separate component, but up to you

Reply:
Opting to use [formatPercentage](https://github.com/getsentry/sentry/blob/f49f397939aed0ae38c377a32be63988fad4609f/static/app/utils/number/formatPercentage.tsx) instead